google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineEnvironment.java

```diff
@@ -23,15 +23,25 @@
  */
 public enum AppEngineEnvironment {
   APP_ENGINE_STANDARD("appengine.environment.name.standard"),
-  APP_ENGINE_FLEX("appengine.environment.name.flexible");
+  APP_ENGINE_FLEX("appengine.environment.name.flexible"),
+  APP_ENGINE_FLEX_COMPAT("appengine.environment.name.flexible");
 
   private String label;
 
   AppEngineEnvironment(String label) {
     this.label = label;
   }
 
+  public boolean isStandard() {
+    return this == APP_ENGINE_STANDARD || this == APP_ENGINE_FLEX_COMPAT;
```

Comment:
I'm glad you called this method out:

Yes, its identified as flex, but its structured like a standard project (has an appengine-web.xml etc.). I'm thinking I might just create a new helper `isFlexCompat()` and just let the callers handle it the way they like - this will be identical to the current logic, but hopefully clearer.